docs/docs/cheat-sheet.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Cheat Sheet
+---
+
+The Gatsby team has created a resource that you might find useful when building a Gatsby site: a cheat sheet with all the top commands and development tips! Feel free to download and print yourself a copy (and tape it by your workstation!). For related online information, visit the [Quick Start](/docs/quick-start/) page.
+
+Get the PDF: <a href="/gatsby-cheat-sheet.pdf" download>gatsby-cheat-sheet.pdf</a>
+
+<figure aria-labelledby="cheat_sheet-text">
+    <h2>Page 1</h2>
+    <a href="/cheat-sheet_page_1.png" title="Click to open image in a new window" target="_blank" style="display:block;">
+        <img src="/cheat-sheet_page_1.png" alt="Cheat Sheet - page 1" style="display:block; margin:0;" />
+    </a>
+    <h2>Page 2</h2>
+    <a href="/cheat-sheet_page_2.png" title="Click to open image in a new window" target="_blank" style="display:block;">
+        <img src="/cheat-sheet_page_2.png" alt="Cheat Sheet - page 2" style="display:block; margin:0;" />
+    </a>
+</figure>
+
+<div id="cheat_sheet-text" style=" position: absolute; height: 1px; width: 1px;overflow: hidden; clip: rect(1px, 1px, 1px, 1px);">
+    <h2>Gatsby Cheat Sheet contents</h2>
+    <p>v1.0 for Gatsby 2.x
+        <a href="https://gatsby.dev/cheatsheet">Latest version <span aria-hidden="true">↗</span></a>
```

Comment:
note: you need to setup this link before publishing this doc

Reply:
💯 — thanks @muescha for the thorough review! 🙏

Reply:
@marcysutton We still need to set up https://gatsby.dev/cheatsheet to point to https://www.gatsbyjs.org/docs/cheat-sheet/. I think @lindaleebumblebee can help with that?

Reply:
@fk @marcysutton @lindaleebumblebee I created the http://gatsby.dev/cheatsheet url

Reply:
🙏 🙏 🙏 Thank you @wardpeet!